assets/js/scan-posts.js

```diff
@@ -31,10 +31,10 @@ const progressPlannerTriggerScan = () => {
 			document.getElementById(
 				'progress-planner-scan-progress'
 			).style.display = 'none';
-			window.location.href = window.location.href.replace(
-				'&content-scan',
-				''
-			);
+
+			window.location.href =
+				window.location.href + '&content-scan-finished=true';
+
```

Comment:
Should this be like this perhaps?
```suggestion

			window.location.href =
				window.location.href.replace(
				'&content-scan',
				''
			) + '&content-scan-finished=true';

```
The current code will not remove the `content-scan` param, should we?

Reply:
I've never seen the `content-scan` in the URL...

Reply:
Yeah, it's just an easter-egg, allowing you to reset and re-scan your content 😉